### ISSUE TITLE
[T9][Accessibility][Functional]:Unable to make selection in familly tab -Pediatric- Expand and collapse

### ISSUE THREAD

**Issue by SR027213** [SR027213/Feb-19-Bug-Bash#33]
steps to reproduce:
Unable to make selection in familly tab -Behavioral - Expand and collapse
